mslib/msui/wms_control.py

```diff
@@ -660,9 +686,13 @@ def get_capabilities(self):
         else:
             # url shortener url translated
             url = request.url
-            url = url.replace('?service=WMS&request=GetCapabilities&version=1.1.1', '')
+
+            # Take the default version of the server, 1.1.1 if not supported, None to determine on initialise_wms
+            version = determine_wms_version(request.content)
+
+            url = url.replace('?service=WMS&request=GetCapabilities', '')
```

Comment:
(Not that this was good before, but) is this robust? Do we assemble the URL here such that we know that this will work?

Reply:
You mean if the url will always contain `?service=WMS&request=GetCapabilities`?
Short answer is no. The safest way remove the appended params would be something like
```
url.replace("?service=WMS", "") \
   .replace("&service=WMS", "") \
   .replace("?request=GetCapabilities", "") \
   .replace("&request=GetCapabilities", "")
```